14/README.md

```diff
@@ -0,0 +1,173 @@
+
+## Goals
+This specification will describe the Tango Logging Service (TLS). 
+
+It will describe and specify such things as:
+- how logging is statically configured for a Device, that is, configured from Device Properties for start up.
+- some logging terms: level, target, rft and path.
+- what are logging targets: console, file, device
+- what is the logging environment variable TANGO_LOG_PATH
+- logging levels: OFF, FATAL, ERROR, WARNING, INFO, DEBUG.
+- logging commands available to all Devices
+- the standard Tango Log Consumer Device and its interface (static mode, dynamic mode).
+
+## Use Cases
+The Tango Logging Service (TLS) provides the programmer with a convenient way to print information which helps to:
+- debug the software
+- report on errors
+- give regular information to user
+- provide timely information of the activities of the Device
+- report the version of the softare
+- provide clues to the internal changes happening in the Device
+
+At run-time, the TLS supports such activities as:
+- changing logging verbosity and destination of any Tango Device
+- capture logs to files
+- display log entries on the command line console
+- interactively view log entries as they are produced by a remote Tango Device
+- review logs already written to files
+- make graphical dashboards with such tools as Kibana and Grafana
+
+## Specification
+
+### Logging Properties 
+TLS SHALL use four Device Properties to control Device Server logging at startup.  The Properties may be set during runtime in the normal way.
+
+#### logging_level property
+SHALL controls the initial logging level of a device. It SHALL have the set of possible values: OFF, FATAL, ERROR, WARN, INFO or DEBUG. This property SHALL be overwritten by the verbose command line option (-v).
+
+#### logging_target property
+SHALL give the name of one or more Logging Target (see below), separated by commas.
+
+#### logging_rft property
+SHALL cause the maximum file size written to be the given value.  If a file grows past this size, SHALL be closed, renamed and a new file opened.
+
+#### logging_path property
+SHALL override the TANGO_LOG_PATH environment variable and SHALL only apply to classes derived from the DServer class.
+The value SHALL specify where log files are written for Logging Target "file" (with no filename).
+
+### Logging Targets
+TLS SHALL allow Logging Targets, that is, destinations, for messages.  A Device Server SHALL emit messages to all targets (comma-separated) to its logging_target Property.
+#### console
+output SHALL be delivered to stdout
+#### file
+output SHALL be written to a standard location in a directory specified by TANGO_LOG_PATH if nonblank, otherwise in a directory created under /tmp/tango-*.
+#### file::filename
+output SHALL be written to the filename given.  File format SHALL be log4j compatible.
```

Comment:
Well, I would say "TLS MUST inform client about the log message format" so we can do so n Tango V10